src/main/java/org/cryptomator/common/LocationPreset.java

```diff
@@ -0,0 +1,60 @@
+package org.cryptomator.common;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Enum of common cloud providers and their default local storage location path.
+ */
+public enum LocationPreset {
+
+	DROPBOX("Dropbox", "~/Dropbox"),
+	ICLOUD("iCloud Drive", "~/Library/Mobile Documents/com~apple~CloudDocs", "~/iCloudDrive"),
```

Comment:
It was called `ICLOUDDRIVE_LOCATIONS` before. Maybe call it `ICLOUDDRIVE`? Would be more consistent with the naming in other places.

Reply:
Changed in [5832e8c](https://github.com/cryptomator/cryptomator/pull/2245/commits/5832e8cd49c516221430c2e4bc969abaf8a4730a)